packages/context/src/lib/context-manager/shared/map-context.directive.ts

```diff
@@ -59,9 +59,18 @@ export class MapContextDirective implements OnInit, OnDestroy {
     // this.component.map.ol.setTarget(target);
 
     const viewContext: ContextMapView = context.map.view;
-    if (!this.component.view || viewContext.keepCurrentView !== true || context.map.view.projection !== this.map.projection) {
-      this.component.view = viewContext as MapViewOptions;
+    // set map view first load
+    if (!this.component.view) {
+      this.component.view = viewContext;
+    } else {
+      if (
+        context.map.view.projection !== this.map.projection ||
+        viewContext.keepCurrentView === true
+      ) {
+        this.component.map.setView(viewContext);
+      }
     }
```

Comment:
Ça me semble l'équivalent comme code, je ne vois pas la différence avec le code précédent?
On ajoute cette ligne `this.component.map.setView(viewContext);`  mais au final `this.component.view = viewContext;` fait la même chose.

Il faudrait voir si c'est la méthode `updateView` qui cause problème dans le fichier `MapBrowserComponent`